src/py/_pyodide/_core_docs.py

```diff
@@ -125,15 +125,46 @@ def typeof(self) -> str:
         return "object"
 
     def object_entries(self) -> "JsProxy":
-        "The JavaScript API ``Object.entries(object)``"
+        """
+        The JavaScript API ``Object.entries(object)``
+
+        Examples
+        --------
+        >>> from pyodide.code import run_js
+        >>> js_obj = run_js("({first: 'aa', second: 22})")
+        >>> entries = js_obj.object_entries()
+        >>> [(key, val) for key, val in entries]
```

Comment:
Wouldn't  `list(entries)` produce the same thing?

Reply:
I think the inner level will still be arrays then.